packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -0,0 +1,185 @@
+  let items: string[] = [""];
+
+  properties
+    ?.filter((property: any) => !isNistNamespace(property?.ns))
+    .map((property: any) => {
+      if (!items.includes(property?.ns)) {
+        items.push(property?.ns);
+      }
+    });
+
+  return items;
+}
+
+interface OSCALPropertiesTableProps {
+  /**
+   * Contains properties elements
+   */
+  properties: Property[];
+  /**
+   * A namespace used for the title of a table
+   */
+  namespace?: string;
+}
+
+const OSCALPropertiesTable = (props: OSCALPropertiesTableProps): ReactElement => {
+  const { properties, namespace } = props;
+
+  return (
+    <>
+      <OSCALSystemImplementationTableTitle variant="h6" id={`${namespace}-table-title`}>
+        {namespace}
+      </OSCALSystemImplementationTableTitle>
+      <TableContainer sx={{ maxHeight: "25em" }}>
+        <Table aria-label="Components" sx={{ height: "max-content" }}>
+          <StyledTableHead>
+            <TableRow>
+              <StyledHeaderTableCell>Name</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Class</StyledHeaderTableCell>
+              <StyledHeaderTableCell>Value</StyledHeaderTableCell>
+            </TableRow>
+          </StyledTableHead>
+          <TableBody>
+            {properties
+              ?.filter((property: any) =>
+                namespace === NIST_NAMESPACE_DOMAIN
+                  ? isNistNamespace(property?.ns)
+                  : property?.ns === namespace
+              )
+              .map((property: any) => (
+                <StyledTooltip title={property?.remarks ?? ""}>
+                  <StyledTableRow key={property?.uuid}>
+                    <TableCell>{property?.name}</TableCell>
+                    <TableCell>{property?.class}</TableCell>
+                    <TableCell>{property?.value}</TableCell>
+                  </StyledTableRow>
+                </StyledTooltip>
+              ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
+};
+
+interface OSCALPropertiesProps {
+  /**
+   * Contains properties elements
+   */
+  properties: any;
+  /**
+   * Title of the element the properties resides under
+   */
+  title: string;
+}
+
+export const OSCALProperties = (props: OSCALPropertiesProps) => {
+  const { properties, title } = props;
+  const thirdPartyNamespaces = getThirdPartyNamespaces(properties);
+
+  return properties ? (
+    <OSCALPropertiesDialog title={title}>
+      <DialogTitle id="scroll-dialog-title">{title} Properties</DialogTitle>
+      {/* Handle NIST properties */}
+      <OSCALPropertiesTable properties={properties} namespace={NIST_NAMESPACE_DOMAIN} key={NIST_NAMESPACE_DOMAIN} />
+      {
+        /* Handle 3rd party properties */
+        thirdPartyNamespaces
+          ?.filter((namespace) => namespace !== "")
+          .map((namespace: any) => (
+            <OSCALPropertiesTable properties={properties} namespace={namespace} key={namespace} />
+          ))
+      }
+    </OSCALPropertiesDialog>
```

Comment:
Let's try to sort to put the `https://csrc.nist.gov/ns/oscal` namespace at the top; however, otherwise it should be treated the exact same as other namespaces. Including using the _value_ (not some other string) in the display.